website/content/en/docs/Installation/_index.md

```diff
@@ -56,5 +56,18 @@ cd lima
 make
 make install
 ```
+
+To change the build configuration, run `make config` or `make menuconfig`.
+
+This requires kconfig tools installed, it is also possible to edit `.config`.
+The default configuration can be found in the file `config.mk` (make syntax).
+
```

Comment:
This paragraphs should be inside `## Kconfig tools`

Reply:
It was for the user that doesn't have any tools, but edits the config as text